pb_chime5/scripts/kaldi_run_rttm_libri_css.py

```diff
@@ -0,0 +1,247 @@
+def my_test_rttm():
+    database_rttm = '/scratch/hpc-prf-nt1/cbj/net/vol/boeddeker/chime6/kaldi/egs/chime6/s5_track2_download/data/dev_beamformit_dereverb_stats_seg/rttm.U06'
+
+
+# get_enhancer = experiment.capture(get_enhancer)
+
+@experiment.capture
+def get_sessions(database_rttm):
+    database_rttm = Path(database_rttm)
+    database_rttm_text = database_rttm.read_text()
+    sessions: set = {line.split()[1] for line in database_rttm_text.splitlines()}
+    return sessions
+
+@experiment.capture
+def get_enhancer(
+    database_rttm,
+    activity_rttm,
+
+    # session_to_audio_paths: A file with a mapping from session ID (or file ID) used
+    #                   in rttm to the actual audio file.
+    # e.g. session_to_audio_paths = {
+    #          session_1: multichannel.wav,
+    #          session_1: [ch1.wav, ch2.wav, ...],
+    # }
+    # Supported are json and yaml.
+    session_to_audio_paths=None,
+
+    # chime6_dir='/net/fastdb/chime6/CHiME6',
+    # multiarray='outer_array_mics',
+    context_samples=240000,
+
+    wpe=True,
+    wpe_tabs=10,
+    wpe_delay=2,
+    wpe_iterations=3,
+    wpe_psd_context=0,
+
+    activity_garbage_class=True,
+
+    stft_size=1024,
+    stft_shift=256,
+    stft_fading=True,
+
+    bss_iterations=20,
+    bss_iterations_post=1,
+
+    bf_drop_context=True,
+
+    bf='mvdrSouden_ban',
+    postfilter=None,
+):
+    from pb_chime5.core_chime6_rttm import (
+        Enhancer,
+        get_database,
+        WPE,
+        Activity,
+        GSS,
+        Beamformer,
+        RTTMDatabase,
+    )
+    if session_to_audio_paths is None:
+        from paderbox.array import intervall as array_intervall
+
+        sessions: set = get_sessions(database_rttm)
+        assert len(sessions) == 1, sessions
+
+        files = list(Path(database_rttm).parent.glob('*.wav'))
+        if len(files) == 1:
+            # Assume multichannel file
+            files, = files
+
+        session_to_audio_paths = {
+            sessions.pop(): files
+        }
+    elif isinstance(session_to_audio_paths, str):
+        import paderbox as pb
+        # Load file, load detects yaml, json, ... (pkl is not allowed -> unsafe)
+        session_to_audio_paths = pb.io.load(session_to_audio_paths)
+    elif isinstance(session_to_audio_paths, dict):
+        session_to_audio_paths = session_to_audio_paths
+    else:
+        raise NotImplementedError(type(session_to_audio_paths), session_to_audio_paths)
+
+    assert wpe is True or wpe is False, wpe
+
+    class MyRTTMDatabase(RTTMDatabase):
+        @staticmethod
+        def example_id(file_id, speaker_id, start, end):
+            # Don't use the strange CHiME-6 pattern for the example ID.
+            max_digits = len(str(16000 * 60 * 60 * 10))  # 10h
+            start = str(start).zfill(max_digits)
+            end = str(end).zfill(max_digits)
+
+            # return f'{file_id}_{speaker_id}-{start}_{end}'
+            return f'{speaker_id}_{start}-{end}'
```

Comment:
Here you could change the file names. The code will save the utterances as `{storage_dir}/audio/{example_id}.wav`